### DISCUSSION TITLE
Make WPH visible on the menu?

### DISCUSSION THREAD

**Discussion by Whydidthechickencrosstheroad** [monkeytypegame/monkeytype#5268] [Ideas] [answered]
The WPH is not on this menu but wouldn't it be useful for someone? Picture that is not on here and picture when on the menu (when you press escape key then type in "unit"). First image: ![Screenshot 2024-03-31 8 57 28 AM](https://github.com/monkeytypegame/monkeytype/assets/158355086/93e3a54b-ab40-4ef8-869b-f92d095c5f6f)

2nd image (captured in the same date as the first picture): 
![Screenshot 2024-03-31 8 58 37 AM](https://github.com/monkeytypegame/monkeytype/assets/158355086/48601c5e-6158-4bfa-96fc-af7b5de63c80)

**Comment by Miodec** [comment, marked as answer]
WPH is meant to be more of a joke thats why its hidden from the settings page.

**Reply by Whydidthechickencrosstheroad** [reply]
Oh, I see
